python/lsst/ts/watcher/rules/cp_verify_alarm.py

```diff
@@ -0,0 +1,266 @@
+
+    Set alarm severity NONE if the cp_verify tests passed,
+    and issue a SERIOUS alarm if not.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"CpVerifyAlarm.{calibration_type}:{ocps_index}",
+    where ``calibration_type`` and ``ocps_index`` are derived from
+    ``config.calibration_type`` and ``config.ocps_index``, respectively.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        remote_index = config.ocps_index
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyAlarm.{remote_info.name}:{remote_info.index}",
+            remote_info_list=[remote_info],
+        )
+
+        self.calibration_type = config.calibration_type
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                calibration_type:
+                    description: >-
+                        Alarm calibration type: BIAS, DARK, or FLAT.
+                    type: string
+                ocps_index:
+                    OCPS index (e.g., OCPS:1: LATISS; OCPS:2: LSSTComCam).
+                verification_threshold:
+                    type: integer
+                    descriptor: Maximum number of failures per detector per test type.
+                default: 8
+            required: [name]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def __call__(self, topic_callback):
+        msg = topic_callback.get()
+        # OCPS response after calling the pipetask
+        response = json.loads(msg.result)
+        # boolean: did verification fail?
+        verify_tests_pass = await self.get_verify_tests_pass_boolean(response)
+
+        if verify_tests_pass:
+            return watcher.NoneNoReason
+        else:
+            # Tests did not pass. Follow up on images and calibrations is
+            # recommended.
+            return AlarmSeverity.SERIOUS, "FAULT state"
+
+    async def get_verify_tests_pass_boolean(self, response_verify):
+        """Determine if cp_verify bias tests passed from OCPS response.
+
+        Parameters
+        ----------
+        response : `dict`
+            OCPS call response.
+
+        Returns
+        -------
+        verify_pass : `bool`
+            Did the cp_verify tests pass?
+        """
+        verify_pass = True
+        # Loop over the entries of the 'results' list
+        # and look for the adecuate dataset type.
+        for entry in response_verify["results"]:
```

Comment:
So, the tricky part here is that if you return an ok here, it means an unrelated OCPS run will reset the alarm level. But I think the only thing that would reset the alarm level is another OCPS run that passes verification. 

I think what you will need to know is keep track of the current alarm level and if an OCPS runs comes through that is not intended for this alarm, you just return the current level/reason.. 

Another alternative would be to raise an exception when this occurs, but this means we will  get an exception every time someone runs an unrelated OCPS job. Not a big deal but maybe not ideal as well..